build-plugin/action.yml

```
@@ -81,6 +81,29 @@ runs:
         printf "plugin-id=%s\n" "${{ steps.package-plugin.outputs.plugin-id }}"
         printf "plugin-version=%s\n" "${{ steps.package-plugin.outputs.plugin-version }}"
 
+    - name: Checkout code
+      if: ${{ env.USE_CHANGELOG_GENERATOR }}
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        ref: main
+
+    - name: Generate Changelog
+      id: github-changelog-generator
+      if: ${{ env.USE_CHANGELOG_GENERATOR }}
+      uses: janheinrichmerker/action-github-changelog-generator@v2.4
+      with:
+        token: ${{ inputs.token }}
+        output: CHANGELOG.md
+        user: ${{ github.repository_owner }}
+        project: ${{ github.event.repository.name }}
+        unreleased: true
+        issues: true
+        pull-requests: true
+        pulls: true
+        include-labels: "bug,enhancement,feature"
+        simple-list: true
```

I'm looking at the inputs here: https://github.com/janheinrichmerker/action-github-changelog-generator?tab=readme-ov-file#inputs

and I think this has to be changed to camelCase, and the `pulls` input doesn't seem to exist:

```suggestion
        pullRequests: true
        includeLabels: "bug,enhancement,feature"
        simpleList: true
```